src/Facebook/PersistentData/FacebookMemoryPersistentDataHandler.php

```diff
@@ -29,28 +29,26 @@
  */
 class FacebookMemoryPersistentDataHandler implements PersistentDataInterface
 {
+    /**
+     * @var array The session data to keep in memory.
+     */
+    protected $sessionData = [];
 
-  /**
-   * @var array The session data to keep in memory.
-   */
-  protected $sessionData = [];
-
-  /**
-   * @inheritdoc
-   */
-  public function get($key)
-  {
-    return isset($this->sessionData[$key])
-      ? $this->sessionData[$key]
-      : null;
-  }
-
-  /**
-   * @inheritdoc
-   */
-  public function set($key, $value)
-  {
-    $this->sessionData[$key] = $value;
-  }
+    /**
+     * @inheritdoc
+     */
+    public function get($key)
+    {
+        return isset($this->sessionData[$key])
+            ? $this->sessionData[$key]
+            : null;
```

Comment:
For me, a ternary is more readable in one line, and it should always be in one line.
If it's too long, it mean you shouldn't use a ternary, but a classic `if`/`else`.
It's my personal tought